src/System.Net.Requests/tests/HttpWebResponseHeaderTest.cs

```diff
@@ -94,6 +94,46 @@ public void HttpContinueMethod(int StatusCode, WebHeaderCollection httpHeaders)
             });
         }
 
+        [OuterLoop]
+        [Fact]
+        public async Task HttpHeader_LastModifiedDate_Success()
+        {
+            DateTime expected = TimeZoneInfo.ConvertTimeFromUtc(new DateTime(2018, 4, 10, 3, 4, 5, DateTimeKind.Utc), TimeZoneInfo.Local);
+            await LoopbackServer.CreateServerAsync(async (server, url) =>
+            {
+                HttpWebRequest request = WebRequest.CreateHttp(url);
+                request.Method = HttpMethod.Get.Method;
+                Task<WebResponse> getResponse = request.GetResponseAsync();
+                await server.AcceptConnectionSendResponseAndCloseAsync(HttpStatusCode.OK, "Last-Modified: Tue, 10 Apr 2018 03:04:05 GMT\r\n", "12345");
+
+                using (WebResponse response = await getResponse)
+                {
+                    HttpWebResponse httpResponse = (HttpWebResponse)response;
+                    Assert.Equal(expected, httpResponse.LastModified);
+                }
+            });
+        }
+
+        [OuterLoop]
+        [Fact]
+        public async Task HttpHeader_LastModifiedDate_Throws()
+        {
+            DateTime expected = TimeZoneInfo.ConvertTimeFromUtc(new DateTime(2018, 4, 10, 3, 4, 5, DateTimeKind.Utc), TimeZoneInfo.Local);
+            await LoopbackServer.CreateServerAsync(async (server, url) =>
+            {
+                HttpWebRequest request = WebRequest.CreateHttp(url);
+                request.Method = HttpMethod.Get.Method;
+                Task<WebResponse> getResponse = request.GetResponseAsync();
+                await server.AcceptConnectionSendResponseAndCloseAsync(HttpStatusCode.OK, "Last-Modified: invalid date\r\n", "12345");
+
+                using (WebResponse response = await getResponse)
```

Comment:
maybe you can optimize this as

```c#
using (HttpWebResponse response = (HttpWebResponse)(await getResponse))
```

etc. Then you wouldn't need to define another local variable.